win32ss/user/user32/windows/draw.c

```diff
@@ -978,6 +978,7 @@ static BOOL UITOOLS95_DrawFrameMenu(HDC dc, LPRECT r, UINT uFlags)
     TCHAR Symbol;
     RECT myr;
     INT cxy;
+    FillRect(dc, r, (HBRUSH)GetStockObject(WHITE_BRUSH));
```

Comment:
Are you sure `WHITE_BRUSH` is the correct brush?

Reply:
Yes, must be white. I confirmed.